docs/get_sql_db.md

```diff
@@ -0,0 +1,43 @@
+# get_sql_db
+
+""" Retrieves summary information for Microsoft SQL databases. Each keyword argument is a query parameter to filter the database details returned i.e. you can query for a specific database name, hostname, instance, is_relic, effective_sla_domain etc.
```

Comment:
`""" Retrieves summary`

We can remove the `"""`

Reply:
thanks